tests/functional/test.cli.run.js

```diff
@@ -47,3 +47,44 @@ describe('web-ext run', () => {
          });
        }));
 });
+
+describe('web-ext run', () => {
```

Comment:
Doesn't [this](https://github.com/mozilla/web-ext/blob/master/tests/unit/test-cmd/test.run.js#L116) test do exactly that?